src/bin.ts

```diff
@@ -149,32 +154,59 @@ yargs
           toVersion,
         });
 
+      const fileToMigrate: ContentRecord = await fs.readJSON(inputPath);
+
+      const servers: {
+        [serverKey: string]: {
+          dataModel: DataModel<"raw">;
+          url: string;
+        };
+      } & { contentServerVersion?: string } = await fs.readJSON(serversPath);
+
       if (fileExtension === "JSON") {
-        // Ensure that Atoti versions are not used as versions to migrate content server
-        migrateContentServer({
-          inputPath,
-          outputPath,
-          serversPath,
+        const { counters, errorReport } = await migrateContentServer({
+          contentServer: fileToMigrate,
+          servers,
           fromVersion: validFromVersion,
           toVersion: validToVersion,
-          removeWidgets,
-          debug,
+          keysOfWidgetPluginsToRemove: removeWidgets,
           doesReportIncludeStacks,
-          onError,
+          behaviorOnError: onError,
```

Comment:
Out of scope, since this problem existed before this PR, but this renaming of variable is an easily avoidable obstacle to debugging, because it means that from a function to the next, the variable is named one thing or another, creating an artificial difficulty during debugging.

![image](https://github.com/activeviam/atoti-ui-migration/assets/5601730/45d9958e-9b22-4435-a2a6-e258dc26867b)

Here, I would rename everything either `onError` or `behaviorOnError`.
But no need to do anything about it in this PR.

Reply:
Yes I completely agree.